<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EXE Map</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QToolButton" name="openButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Open ROM</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Save ROM</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="mapListGroupBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Map List</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeWidget" name="mapTreeWidget">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="mapTab">
         <attribute name="title">
          <string>Map</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QGraphicsView" name="graphicsView">
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>0</height>
                </size>
               </property>
               <property name="backgroundBrush">
                <brush brushstyle="CrossPattern">
                 <color alpha="255">
                  <red>150</red>
                  <green>200</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Tile Set</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QTableWidget" name="tilesetTable">
                    <property name="showGrid">
                     <bool>false</bool>
                    </property>
                    <property name="rowCount">
                     <number>64</number>
                    </property>
                    <property name="columnCount">
                     <number>16</number>
                    </property>
                    <attribute name="horizontalHeaderVisible">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="horizontalHeaderDefaultSectionSize">
                     <number>16</number>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="verticalHeaderDefaultSectionSize">
                     <number>16</number>
                    </attribute>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Palette</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QTableWidget" name="backgroundPaletteTable">
                    <property name="rowCount">
                     <number>16</number>
                    </property>
                    <property name="columnCount">
                     <number>16</number>
                    </property>
                    <attribute name="horizontalHeaderDefaultSectionSize">
                     <number>20</number>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                     <bool>true</bool>
                    </attribute>
                    <attribute name="verticalHeaderDefaultSectionSize">
                     <number>20</number>
                    </attribute>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <row/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                    <column/>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="movementTab">
         <attribute name="title">
          <string>Movement</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGraphicsView" name="graphicsView_2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
